examples/ios/swift-2.2/RealmExamples.xcodeproj/project.pbxproj

```diff
@@ -1089,6 +1141,56 @@
 			};
 			name = Release;
 		};
+		E8F14D0F1CF8FD0C00564AF5 /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				CLANG_ANALYZER_NONNULL = YES;
```

Comment:
I'd assumed there were more that were duplicates. Out of the others that aren't duplicates, a lot are just unnecessary. I wouldn't bother spending any more time tidying this up though given how obscure this new target is.